documentation/modules/creating-migration-quick-plan-existing-source.adoc

```diff
@@ -22,7 +24,28 @@ The *Create migration plan* pane opens. It displays the source provider's name a
 . Click *Add mapping* to edit a suggested network mapping or a storage mapping, or to add one or more additional mappings.
 . Click *Create migration plan*.
 +
-{project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} again validates the plan.
+{project-short} validates the migration plan and the *Plan details* page opens, indicating whether the plan is ready for use or contains an error. In all cases, the details of the plan are listed, and you can edit the items you filled in on the previous page. If you make any changes, {project-short} again validates the plan
+
+. VMware source providers only (Both optional):
+
+.. Preserving the CPU type of VMs that are migrated from {rhv-short}: By default, {project-short} sets the CPU type (CPU model and any flags) of migrated VMs if the type is not set specifically for the VM. This can be an issue for migrations from {rhv-short} because the CPU type of {rhv-short} VMs can be set at the cluster level, in which case {project-short} might change the type.
++
+To preserve the CPU type of your {rhv-short} VMs, in the *Settings* section, click the Edit icon next to *Preserve CPU model*. toggle the *Whether to preserve the CPU model* switch, and then click *Save*.
+
+.. Specifying a root device: Applies to multi-boot VM migrations only. By default, {project-short} uses the first bootable device detected as the root device.
++
+To specify a different root device, in the *Settings* section, click the Edit icon next to *Root device* and enter the root device in the *Edit root device* text box, using the following format: `/dev/sdX#`, where X represents the disk number in letters (that is the first disk would be represented as 'a', the second disk as 'b', and so on) and # represents the partition that holds the operating system inside the disk (starting from 1). For example, to set the second disk as the root device and to specify that the operating system is on the disk's second partition, enter the following input: `/dev/sdb2`. After you enter the boot device, click *Save*.
```

Comment:
We have drop down menu for the common options in the UI and also an option to free text, so its should say something like  `select the desired option from the menu, in case the option is not presented please provide the following format input....`

Reply:
@bkhizgiy @yaacov  I saw the drop-down list but i thought it just included examples that weren't connected to the specific provider. Will the choices listed be "likely guesses" for that specific migration or will they be the same in all cases?

Reply:
> Will the choices listed be "likely guesses" for that specific migration or will they be the same in all cases?

we have a drop-down for the most common options for all cases, the are not specific for this migration.

> I saw the drop-down list but i thought it just included examples

The drop down show most common cases, they are the "likely guesses" for any migration. If a user need a different directory, they can write a different option that is not one of the suggested options in a text field.

Reply:
p.s.

current text in the UI:

https://github.com/kubev2v/forklift-console-plugin/blob/09c3deec54c639ad17976f8f2eb8deb7b928fa1c/packages/forklift-console-plugin/src/modules/Plans/views/details/components/SettingsSection/modals/EditRootDisk/editRootDiskModalBody.tsx#L14